packages/code-du-travail-frontend-e2e/package.json

```diff
@@ -3,26 +3,18 @@
   "version": "4.91.1",
   "private": true,
   "description": "",
-  "main": "codecept/codecept.conf.js",
   "scripts": {
     "precommit": "lint-staged",
-    "lint": "eslint ./**/*.{js,ts}",
-    "test:e2e": "yarn run test:codeceptjs && yarn run test:cypress",
-    "test:cypress": "cypress run",
-    "test:cypress:ui": "cypress open",
-    "test:codeceptjs": "codeceptjs run --features --config=./codecept/codecept.conf.js",
-    "test:codeceptjs:headed": "CODECEPT_HEADED=true codeceptjs run --features  --config=./codecept/codecept.conf.js  --grep \"Parcours en ne connaissant pas sa convention collective\"   ",
-    "test:codeceptjs:ui": "codecept-ui --verbose  --config=./codecept/codecept.conf.js"
+    "lint": "eslint ./**/*.ts",
+    "test:e2e": "cypress run",
+    "test:e2e:ui": "cypress open"
   },
   "dependencies": {
-    "@codeceptjs/helper": "^1.0.2",
-    "@codeceptjs/ui": "^0.4.6",
-    "codeceptjs": "^3.0.7",
-    "puppeteer": "^10.4.0"
   },
   "devDependencies": {
     "@socialgouv/eslint-config-typescript": "^1.93.0",
     "cypress": "12.7.0",
+    "cypress-iframe": "^1.0.1",
```

Comment:
Ajout d'une lib pour gérer l'iframe. J'ai essayé sans la lib mais ça ne fonctionne pas... La doc sur le sujet que j'ai trouvé : https://www.cypress.io/blog/2020/02/12/working-with-iframes-in-cypress (il référence la lib dedans)